### PR TITLE
fix basic example, fixes #363

### LINKED ISSUE #363: "echo $UNICORNS" in "/basics" documentation page
The [basics page](https://devenv.sh/basics/) shows a `echo $UNICORNS` command that is supposed to print "yes". I didn't understand where this could come from so I tried and nothing is printed:

```
$ devenv shell
Building shell ...
Entering shell ...

hello
jq-1.6

$ echo $UNICORNS
$
```

I'm using version 0.5.1.

### PR DESCRIPTION
**Pull request by shyim**
Fixes docs issue #363

### CODE REVIEW

File: docs/basics.md [hunk 1]
```diff
@@ -29,11 +29,8 @@ $ devenv shell
 Building shell ...
 Entering shell ...
 
+(devenv) $ echo $GREET
 hello
-jq-1.6
```
Comment (sandydoo) [inline docs/basics.md:33]: It'd still have to be:
```
$ devenv shell
Building shell ...
Entering shell ...

hello 
jq-1.6

(devenv) $ echo $GREET
hello
```

I really have no idea what @domenkozar was trying to tell us about the unicorns 😂

**Review by sandydoo — CHANGES_REQUESTED** [review]

File: docs/basics.md (hunk 1 above)
Comment (shyim) [reply docs/basics.md:33]: It makes for me no sense to show the packages or what it should be 🤔

Comment (sandydoo) [reply docs/basics.md:33]: My point was that the output should match the previous example.

Example: 
```nix
{ pkgs, ... }: # (1)!

{ # (2)!
  env.GREET = "hello"; # (3)!

  packages = [ pkgs.jq ];

  enterShell = ''
    echo $GREET
    jq --version
  ''; # (4)!
}
```
Output:
```
$ devenv shell
Building shell ...
Entering shell ...

hello 
jq-1.6

(devenv) $ echo $GREET
hello
```

> It makes for me no sense to show the packages or what it should be 🤔
I think the idea is to show how to add packages and that they then become available in the shell.

Comment (shyim) [reply docs/basics.md:33]: is updated :)

**Pull request merged**